AIM/AIM/UI/Views/TrackListViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  TrackListViewController.swift
+//  AIM
+//
+//  Created by Oskar Szydlowski on 28.02.2018.
+//  Copyright © 2018 Oskar Szydlowski. All rights reserved.
+//
+
+import UIKit
+
+class TrackListViewController: UIViewController, UITableViewDelegate, UITableViewDataSource, TrackListView {
+    
+    private let presenter: TrackListPresenter
+    
+    private let tableView = UITableView()
+    
+    private let rowHeight: CGFloat = 90.0
+    
+    init(presenter: TrackListPresenter) {
+        self.presenter = presenter
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupTableView()
+        fetchOnAirInfo()
+    }
+    
+    private func setupTableView() {
+        view.addSubview(tableView, with: [
+            tableView.topAnchor.constraint(equalTo: view.topAnchor),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+        ])
+        
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.register(TrackCell.self)        
+    }
+    
+    // MARK: - UITableViewDelegate
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return rowHeight
+    }
+    
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        guard let _ = presenter.onAirInfo?.stationInfo else {
```

Comment:
view shouldn't 'inspect' presenter like that. This breaks law of Demeter. Presenter should provide this with some variable or method, e.g. `isHeaderVisible` or sth similiar